### PR TITLE
Update circleci configuration

### PR DESCRIPTION
**Pull request by silvin-lubecki**
* Add master branch build status badge to the main README page
* Bump golang version to `1.13.7`

image

### CODE REVIEW

**Comment by carolynvs** [comment]
Does circle need to be reconfigured to know about the new location of the repository?

**Comment by silvin-lubecki** [comment]
That's what I thought too @carolynvs but I think the issue here with this circleci check that is continuously waiting is that circle ci is not approved by the `cnabio` organization 🤔 
That said, with @radu-matei we think that we should move to github actions, but maybe that's something we should decide at an organization level so every repository would have the same CI structure? We can discuss that point at the next CNAB community meeting. 👍

**Comment by carolynvs** [comment]
I am happy to approve circle until we get it figured out, no need to go without CI while we decide. Do you want to send a request? I just checked at the org level and didn't see anything pending.

**Comment by radu-matei** [comment]
If enabling Circle for the organization is not a big decision, then I'd be much more comfortable doing that until we can have a community decision about switching CI systems.
(I am definitely in favor of moving to GH actions though).

**Comment by radu-matei** [comment]
It turns out Circle is actually running the builds (since the repo was moved), but it is not reporting the results back to GitHub (probably because it is not authorized) - https://circleci.com/gh/cnabio/cnab-to-oci/300

**Comment by vdice** [comment]
When I navigate to the org settings according to https://circleci.com/docs/2.0/enable-checks/, I encounter the following error: 
Screen Shot 2020-02-14 at 9 05 38 AM

**Comment by radu-matei** [comment]
I see the same error as @vdice here.
This might already be more of a headache than just using something already enabled..

**Comment by vdice** [comment]
Oh, when I refreshed the page, it showed the enable button, etc.  I believe it is added now (currently just to this one repo.)  Try with a new push @silvin-lubecki ?

**Comment by silvin-lubecki** [comment]
Supersed by #97 , rebased so now just bumping golang and adding circle ci badge, PTAL 🙏

**Review by bin-ext-windows — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-ARG ALPINE_VERSION=3.10
-ARG GO_VERSION=1.13.0
+ARG ALPINE_VERSION=3.11
+ARG GO_VERSION=1.13.7
 
 # build image
 FROM golang:${GO_VERSION}-alpine${ALPINE_VERSION} as build
```
Comment (bin-ext-windows) [inline Dockerfile:5]: b70d986af6dd0adf70287a23f260c652560f09bb

**Review by bin-ext-windows — APPROVED** [review]

**Pull request merged**